.openshift-ci/build/generate-db-dump.sh

```diff
@@ -12,10 +12,10 @@ gate_job generate-db-dump
 generate_db_dump() {
     info "Generating DB dump"
 
-    groupadd -g 71 pg
-    adduser pg -u 71 -g 71 -d /var/lib/postgresql -s /bin/sh
+    groupadd -g 1001 pg
```

Comment:
Although, why don't we simply use the next available ID or use [`-r` (`--system`)](https://man7.org/linux/man-pages/man8/groupadd.8.html)?

Reply:
Keeping it a system GID/UID might be slightly better because GN/Linux distros sometimes have configurations in place that only affects one of the types, like being more permissive for users. Not sure if this would be relevant in a container env, though.

Reply:
I didn't know that existed :) that being said, I think being explicit is ok, as it gives us full control of these values. Also, I don't want to rely on the value of `USERGROUPS_ENAB`, as I don't know what it is and if it could change if we update the base image, so I don't want something like `adduser` to create a new group for us https://linux.die.net/man/8/adduser

Reply:
Interesting. How changes in the UID when rebuilding the base image could affect us?

Reply:
I think it could be useful for debugging purposes. Also, I don't think there is a concise way to get the group ID given a group name https://askubuntu.com/questions/639990/what-is-the-group-id-of-this-group-name. Might as well just set one ourselves

Reply:
Humm, OK about debugging.

Anyway, you don't have to worry about `getent` unless on NSS, simply parsing `/etc/groups` would do it.